### ISSUE TITLE
ESP32 board

### ISSUE THREAD

**Issue by silviudc** [WhoTheHeck/ESP8266-Multisensor#4]
Any chance to update this to accommodate the 30 pin esp32 boards?
Like these ones: https://smile.amazon.co.uk/gp/product/B0814WGPVY/ref=ppx_yo_dt_b_asin_title_o02_s00?ie=UTF8&psc=1
Super handy for using the new BT proxy integrations.

**Comment by WhoTheHeck** [comment]
An ESP32 board would indeed be a nice upgrade since these also seem to have a more stable wifi connection besides the BT benefit. Unfortunetly the pinouts are too different for a quick fix. The kicad_pcb file in the PCB folder would have to be rerouted.

IMO a new ESP32-based board with more and better sensors (e.g. a better CCS811 alternative) would be preferable.

**Comment by silviudc** [comment]
Yes and no with the ESP32 and BT proxy, it seems like BT proxy uses most of the memory of the ESP32 so with this many sensors might not even work. 
For example BT proxy and an ILI9341 screen just doesn't work, either one or the other but not both.
Might be ok for temp/hum/lux/motion sensors and BT proxy but not guaranteed.

**Comment by silviudc** [comment]
Decided to add make a bit of a HAT for the sensors, in case any of them fail, unplug and replace
![image](https://user-images.githubusercontent.com/17740184/199937447-721bcc98-15f5-4412-a1ec-da9ed3520ec3.png)
![image](https://user-images.githubusercontent.com/17740184/199937569-5733a76f-571b-45d2-98c8-17e3e3fae687.png)
![image](https://user-images.githubusercontent.com/17740184/199937655-46e53cf3-f205-4856-95a8-285ec002450c.png)
Still fits in the original STL but need to work out something for the back cover

**Comment by WhoTheHeck** [comment]
nice to see another one is made!
not sure if one of the sensors ever will break, but also for disassembly it is definetly a good idea.
I could adjust the model to accomodate for the additonal length. Basically just make it a little deeper.

**Comment by WhoTheHeck** [comment]
> Yes and no with the ESP32 and BT proxy, it seems like BT proxy uses most of the memory of the ESP32 so with this many sensors might not even work. For example BT proxy and an ILI9341 screen just doesn't work, either one or the other but not both. Might be ok for temp/hum/lux/motion sensors and BT proxy but not guaranteed.

I realized that issue with an ESP32-Cam module the other day too, but thought it might be due to the additonal camera stream. Not sure if any of the ESP32's would have enough capability to fulfill all the functions then.

**Comment by silviudc** [comment]
I have seen temp&hum, maybe co2 sensors that can fail but at the same time all the parts are cheap enough (until you add them up ;))
Other thing I was thinking of was ventilation but the esp8266 doesn't get as hot as the esp32 so maybe ok, still wouldn't hurt to have a few holes on the back cover.
